aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
```

Comment:
"in process" and "out of process" throughout, or hyphenate if you prefer, main thing is to be consistent within the doc.